src/components/InlineActionsContainer.vue

```diff
@@ -19,7 +19,7 @@
   -->
 <template>
 	<li class="inline-container-base">
-		<ul>
+		<ul class="inline-container-content">
 			<slot />
 		</ul>
 	</li>
```

Comment:
Could we have that component exposed and documented? :)

Reply:
Submitting to nextcloud vue components?

Reply:
Yes or something like that